test/fixtures/scale.radialLinear/anglelines-reverse-scale.js

```diff
@@ -0,0 +1,29 @@
+module.exports = {
+  config: {
+    type: 'radar',
+    data: {
+      labels: ['A', 'B', 'C', 'D', 'E']
+    },
+    options: {
+      responsive: false,
+      scales: {
+        r: {
+          reverse: true,
+          grid: {
+            display: true,
+          },
+          angleLines: {
+            color: 'red',
+            lineWidth: 5,
+          },
+          pointLabels: {
+            display: false
+          },
+          ticks: {
+            display: false
```

Comment:
We need to enable these, in case the property gets moved this test will still be positive